<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>BehaviourSetup</class>
 <widget class="QWidget" name="BehaviourSetup">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>411</width>
    <height>493</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>20</y>
     <width>351</width>
     <height>51</height>
    </rect>
   </property>
   <property name="title">
    <string>模式选择</string>
   </property>
   <widget class="QRadioButton" name="proxyRBtn">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>89</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>代理模式</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="serverRBtn">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>20</y>
      <width>89</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>服务器模式</string>
    </property>
   </widget>
   <widget class="QPushButton" name="configBtn">
    <property name="geometry">
     <rect>
      <x>274</x>
      <y>16</y>
      <width>61</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>文件配置</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_3">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>80</y>
     <width>351</width>
     <height>211</height>
    </rect>
   </property>
   <property name="title">
    <string>其他参量监测</string>
   </property>
   <widget class="QCheckBox" name="batteryCBtn">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>90</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>电池温度</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="cpuTempCBtn">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>120</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>CPU温度</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="batteryLineEdit">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>90</y>
      <width>71</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="cpuTempLineEdit">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>120</y>
      <width>71</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="batteryLabel">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>90</y>
      <width>61</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>警戒值(M)：</string>
    </property>
   </widget>
   <widget class="QLabel" name="cpuTempLabel">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>120</y>
      <width>61</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>警戒值(%)：</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="wifiCBtn">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>150</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Wifi流量</string>
    </property>
   </widget>
   <widget class="QLabel" name="wifiLabel">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>150</y>
      <width>81</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>警戒值（M）：</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="wifiLineEdit">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>150</y>
      <width>71</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="mobileLabel">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>180</y>
      <width>81</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>警戒值（M）：</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="mobileCBtn">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>180</y>
      <width>81</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Mobile流量</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="mobileLineEdit">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>180</y>
      <width>71</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QCheckBox" name="memCBtn">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>30</y>
      <width>51</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>内存</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="cpuCBtn">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>57</y>
      <width>51</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>CPU</string>
    </property>
   </widget>
   <widget class="QLabel" name="memLabel">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>30</y>
      <width>61</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>警戒值(M)：</string>
    </property>
   </widget>
   <widget class="QLabel" name="cpuLabel">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>60</y>
      <width>61</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>警戒值(%)：</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="memLineEdit">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>30</y>
      <width>71</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="cpuLineEdit">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>60</y>
      <width>71</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="versionGroup">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>300</y>
     <width>351</width>
     <height>51</height>
    </rect>
   </property>
   <property name="title">
    <string>APP版本</string>
   </property>
   <widget class="QRadioButton" name="debugRBtn">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>20</y>
      <width>89</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>debug</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="releaseRBtn">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>20</y>
      <width>89</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>release</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_4">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>360</y>
     <width>351</width>
     <height>51</height>
    </rect>
   </property>
   <property name="title">
    <string>IP &amp;&amp; Port</string>
   </property>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>20</y>
      <width>31</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>IP:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="IPLineEdit">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>20</y>
      <width>121</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>20</y>
      <width>31</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Port:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="portLineEdit">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>20</y>
      <width>71</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_2">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>420</y>
     <width>351</width>
     <height>51</height>
    </rect>
   </property>
   <property name="title">
    <string>xml文件根路径</string>
   </property>
   <widget class="QLineEdit" name="xmlRootLineEdit">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>251</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="browseBtn">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>20</y>
      <width>51</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>浏览</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
